week-1/Homework/mandatory/5-magic-8-ball.js

```diff
@@ -55,8 +100,52 @@ let answer;
 // - positive
 // - negative
 // - very negative
-function checkAnswer() {}
-
+function checkAnswer() {
+  if (number === 1) {
+    return "verr positive";
+  } else if (number === 2) {
+    return "verr positive";
+  } else if (number === 3) {
+    return "verr positive";
+  } else if (number === 4) {
+    return "verr positive";
+  } else if (number === 5) {
+    return "verr positive";
+  } else if (number === 6) {
+    return "positive";
+  } else if (number === 7) {
+    return "positive";
+  } else if (number === 8) {
+    return "positive";
+  } else if (number === 9) {
+    return "positive";
+  } else if (number === 10) {
+    return "Positive";
+  } else if (number === 11) {
+    return "negative";
+  } else if (number === 12) {
+    return "negative";
+  } else if (number === 13) {
+    return "negative";
+  } else if (number === 14) {
+    return "negative";
+  } else if (number === 15) {
+    return "negative";
+  } else if (number === 16) {
+    return "verr negative";
+  } else if (number === 17) {
+    return "verr negative";
+  } else if (number === 18) {
+    return "verr negative";
+  } else if (number === 19) {
+    return "verr negative";
+  } else if (number === 20) {
+    return "verr negative";
```

Comment:
There's a quicker (and in my opinion more fun) way to do this! 

```
if (number < 6) {
  return "verr positive"
}
else if (number < 11) {
  return "positive"
}
else if ...
```

Much less typing 😁